### PR TITLE
(BLOG) Status update for 2021-01-15

### PR DESCRIPTION
**Pull request by sanfrancrisko**

### CODE REVIEW

**Review by Disha-maker — COMMENTED** [review]
Great work @sanfrancrisko

**Comment by Disha-maker** [comment]
Was tagmail not released this week on Triage day this week, or was it last week @sanfrancrisko. I remember doing a session on tagmail JIRA Ticket with @pmcmaw that's why I asked :)

**Comment by pmcmaw** [comment]
Tagmail was released 10 days ago, we worked on it 2 weeks ago, it was mentioned in last weeks blog @Disha-maker

File: _posts/2021-01-15-status-update.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+## Puppet Camp: Call for Papers (March, April)
+
+Puppet just announced on Thursday the CFPs for the two virtual Puppet Camps in March and April are now live.
+Please feel free to spread the word and of course, if you are interested in submitting a paper, full details [here](https://puppet.com/blog/puppet-camp-call-for-papers-now-open-for-march-and-april/)
+
+## Cloud CI Update
+
+The Cloud CI provisioning service is nearing it's first v1.0.0 release.
+We have begun to tidy up a couple of loose ends for the initial version.
+Although we have over 35 of the IAC supported modules using the service, some of our more complex modules are still not ported.
```
Comment (DavidS) [inline _posts/2021-01-15-status-update.md:47]:
```suggestion
We now have 35 of 47 the IAC supported [modules](https://puppetlabs.github.io/iac/modules/) using the service, only the most complex modules are still not ported, and will require more investment in supporting their additional needs.
```

**Review by DavidS — CHANGES_REQUESTED** [review]

File: _posts/2021-01-15-status-update.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+## DSC Modules Update
+
+As of this week, there are 68 puppetized PowerShell modules with DSC Resources up [on the forge in the dsc namespace][forge-dsc].
+We have meticulously converted all available versions of these modules in case you need a direct move from a prior version of one of the modules, meaning there's 642 total releases available right now!
+We're hard at work on documentation and guidance for troubleshooting these autogenerated modules and transitioning from our legacy DSC integrations, more to follow!
+
+## Inappropriate Terminology Removal
+
+[DavidSwan][DavidSwan] has been diligently continuing with the effort to remove inappropriate terminology from our code repos.
```
Comment (DavidS) [inline _posts/2021-01-15-status-update.md:59]:
```suggestion
[David Swan][DavidSwan] has been diligently continuing with the effort to remove inappropriate terminology from our code repos.
```

**Review by DavidS — APPROVED** [review]

**Pull request merged**